.pipelines/templates/build-steps.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+  # 1. Configure node
+  - task: NodeTool@0
+    displayName: Use Node 20.x
+    inputs:
+      versionSpec: 20.x
```

Comment:
wonder if we should use 18.x because support for 20 was only added in itwinjs 4.3

Reply:
Switched back to 18.16.0